src/common/ui-wrapper.js

```diff
@@ -91,10 +90,9 @@ class UIWrapper {
     this.setConfig({hasError: false}, 'engine');
   }
 
-  setSeekbarConfig(mediaConfig: KPMediaConfig, uiConfig: KPUIOptionsObject): void {
+  setSeekbarConfig(uiConfig: KPUIOptionsObject, thumbnailConfig: ?SeekbarConfig): void {
```

Comment:
this is api (via `kalturaPlayer.ui.setSeekbarConfig`) i'm not sure we can break it

Reply:
It shouldn't be an API and you should have hidden it once you exposed the `ui` namespace. If someone wants to set seekbar config it should be done via player config - this method is internal for us to communicate with the ui manager.
Anyway it will be removed and will be useless once the UI will work with the `getThumbnail` API.